packages/helpers/theme/lineHeights.js

```diff
@@ -0,0 +1,8 @@
+import { theme } from '@chakra-ui/core';
+
+export default {
+  nonePlus1: '19px',
+  nonePlus: '17px',
+  noneMinus: '14px',
+  ...theme.lineHeights,
```

Comment:
Lets put the spread at top to prevent accidental overrides

Reply:
You are correct. While starting I thought of different thing, did not note down to change it.
I forgot, I will change and push a new commit.

Reply:
This is fixed,